package.json

```diff
@@ -27,18 +27,45 @@
     "test": "pnpm lint && pnpm test:types && pnpm vitest run --coverage",
     "test:types": "tsc --noEmit --skipLibCheck"
   },
+  "pnpm": {
+    "overrides": {
+      "@obuild/plugin": "file:plugins/obuild-plugin-0.0.1.tgz",
+      "@obuild/plugin-oxc-transform": "file:plugins/obuild-plugin-oxc-transform-0.0.1.tgz",
+      "@obuild/plugin-oxc-dts": "file:plugins/obuild-plugin-oxc-dts-0.0.1.tgz",
+      "@obuild/plugin-oxc-minify": "file:plugins/obuild-plugin-oxc-minify-0.0.1.tgz",
+      "@obuild/plugin-postcss": "file:plugins/obuild-plugin-postcss-0.0.1.tgz",
+      "@obuild/plugin-sass": "file:plugins/obuild-plugin-sass-0.0.1.tgz",
+      "@obuild/plugin-shared": "file:plugins/obuild-plugin-shared-0.0.1.tgz",
+      "@obuild/plugin-tsc-dts": "file:plugins/obuild-plugin-tsc-dts-0.0.1.tgz",
+      "@obuild/plugin-tsgo-dts": "file:plugins/obuild-plugin-tsgo-dts-0.0.1.tgz",
+      "@obuild/plugin-vue-tsc-dts": "file:plugins/obuild-plugin-vue-tsc-dts-0.0.1.tgz",
+      "@obuild/plugin-vue-sfc-transformer": "file:plugins/obuild-plugin-vue-sfc-transformer-0.0.1.tgz"
+    }
+  },
   "dependencies": {
+    "@obuild/plugin": "file:plugins/obuild-plugin-0.0.1.tgz",
+    "@obuild/plugin-oxc-transform": "file:plugins/obuild-plugin-oxc-transform-0.0.1.tgz",
+    "@obuild/plugin-oxc-dts": "file:plugins/obuild-plugin-oxc-dts-0.0.1.tgz",
+    "@obuild/plugin-oxc-minify": "file:plugins/obuild-plugin-oxc-minify-0.0.1.tgz",
```

Comment:
These should be built-in: (as of current version)
- plugin core
- oxc transformer, minifier, and dts generator